front/app/components/ContentUploadDisclaimer/messages.ts

```diff
@@ -4,7 +4,7 @@ export default defineMessages({
   contentUploadDisclaimerFull: {
     id: 'app.components.ContentUploadDisclaimer.contentUploadDisclaimerFull2',
```

Comment:
You'll need to change the ID to get the text change to be recognised

Reply:
Indeed, but the translation changes were already done on Wednesday and yesterday, so in this case I think it's actually desired that Crowdin doesn't recognize this. I double checked and we do have "Go Vocal" now in the corresponding entries in the json files.